config/types.go

```diff
@@ -221,3 +221,10 @@ type RawService map[string]interface{}
 
 // RawServiceMap is a collection of RawServices
 type RawServiceMap map[string]RawService
+
+type ParseOptions struct {
```

Comment:
Missing a comment :stuck_out_tongue:.
I also wonder if it would make sence to put it in `project/options` package instead and thus would become `options.Parse`.